src/app/external-cluster-wizard/steps/external-cluster/provider/eks/component.ts

```diff
@@ -295,7 +327,7 @@ export class EKSClusterSettingsComponent
       } as ExternalCloudSpec,
       spec: {
         eksclusterSpec: {
-          roleArn: this.controlValue(Controls.RoleArn),
+          roleArn: this.selectedRoleArn,
```

Comment:
To be consistent on places
this.controlValue(Controls.RoleArn)?.[ComboboxControls.Select]

Reply:
no im using different  value not the one been selected so the list show name and roleArn but on select i show the name in the field but send the arn in the payload that's why im using the `(changed)= onRoleArnChange()`

Reply:
ah! got it previously in wizard we were typing `arn::...` and we aligned with AWS `Cluster service role` that's why. Okay you can leave it. 👍🏽